packaging/android/build.sh

```diff
@@ -90,14 +90,8 @@ elif [ -d "$SUBSURFACE_SOURCE/../Qt/5.9" ] ; then
 	export QT5_ANDROID=$SUBSURFACE_SOURCE/../Qt/5.9
 elif [ -d "$SUBSURFACE_SOURCE/../Qt/5.8" ] ; then
 	export QT5_ANDROID=$SUBSURFACE_SOURCE/../Qt/5.8
-elif [ -d "$SUBSURFACE_SOURCE/../Qt/5.7" ] ; then
-	export QT5_ANDROID=$SUBSURFACE_SOURCE/../Qt/5.7
-elif [ -d "$SUBSURFACE_SOURCE/../Qt/5.6" ] ; then
-	export QT5_ANDROID=$SUBSURFACE_SOURCE/../Qt/5.6
-elif [ -d "$SUBSURFACE_SOURCE/../Qt/5.5" ] ; then
-	export QT5_ANDROID=$SUBSURFACE_SOURCE/../Qt/5.5
```

Comment:
would it be a good idea to mention that you removed support for Qt5.5-5.8

Reply:
Commit message updated.